libs/perun/components/src/lib/members-dynamic-list/members-dynamic-list.component.html

```diff
@@ -34,8 +34,8 @@
             [aria-label]="checkboxLabel(member)"
             [checked]="selection.isSelected(member)"
             attr.data-cy="{{member.user.firstName | lowercase}}-checkbox"
-            [disabled]="member.membershipType === 'INDIRECT'"
-            [matTooltip]="member.membershipType === 'INDIRECT' ? ('MEMBERS_LIST.CHECKBOX_TOOLTIP' | translate):''"
+            [disabled]="member.membershipType === 'INDIRECT' || !(member | memberLifecycleAlterable)"
+            [matTooltip]="checkboxTooltip(member)"
```

Comment:
for tooltip also do pipe
when you are using some method in HTML, the method is called everysecond or something like that to update the text
the pipe is called only when the input parameter Is changed